iotdb-core/datanode/src/main/java/org/apache/iotdb/db/storageengine/dataregion/compaction/execute/utils/executor/fast/element/PageElement.java

```diff
@@ -95,20 +103,84 @@ public PageElement(
     this.pageData = timePageData;
     this.valuePageDatas = valuePageDatas;
     this.priority = priority;
-    this.iChunkReader = alignedChunkReader;
     this.startTime = pageHeader.getStartTime();
     this.chunkMetadataElement = chunkMetadataElement;
     this.isLastPage = isLastPage;
     this.needForceDecoding = chunkMetadataElement.needForceDecoding;
   }
 
   public void deserializePage() throws IOException {
-    if (iChunkReader instanceof AlignedChunkReader) {
-      this.pointReader =
-          ((AlignedChunkReader) iChunkReader)
-              .getPagePointReader(pageHeader, valuePageHeaders, pageData, valuePageDatas);
+    if (this.iChunkReader == null) {
```

Comment:
What does `iChunkReader == null` mean ?

Reply:
The variable 'iChunkReader' was not set during construction when the series of this PageElement is aligned.